backend/upload.js

```diff
@@ -0,0 +1,24 @@
+import multer from 'multer'
+
+var Storage = multer.diskStorage({
+  destination: function (req, file, callback) {
+    callback(null, './images')
```

Comment:
We need more information from the Sysadmins where we're actually going to be storing the images (i.e. if they're going on the same hard-drive as our system or on 3rd-party object storage).

Also, the convention I've seen for directory / file naming is to generate a unique hash and save the file based on that hash. For example:
Uploaded Image: `my_awesome_photo.jpg`
Could be hashed to: `kTK6vi6Z4IAlh65eSMps.jpg`
And would be saved in the directory hierarchy: `k/T/kTK6vi6Z4IAlh65eSMps.jpg`

We'd rename the files when they're downloaded (based on the student's name and submission title). If we needed to keep the original file name, we could store it in the db or append it to the hash (e.g. `kTK6vi6Z4IAlh65eSMps_my_awesome_photo.jpg`)